runhouse/main.py

```diff
@@ -91,7 +91,15 @@ def load_cluster(cluster_name: str):
         c._update_from_sky_status(dryrun=True)
 
 
-def _start_server(restart, restart_ray, screen, create_logfile=True, host=None):
+def _start_server(
+    restart,
+    restart_ray,
+    screen,
+    create_logfile=True,
+    host=None,
```

Comment:
Do we need to add a "port" parameter here and elsewhere in this file, or would this be handled outside of this PR?